application/src/main/java/org/togetherjava/tjbot/config/Config.java

```diff
@@ -342,4 +345,14 @@ public String getSourceCodeBaseUrl() {
     public JShellConfig getJshell() {
         return jshell;
     }
+
+    /**
+     * Gets the config for the Starboard Contains the List of emoji names recognized by the
+     * starboard as well as the name of the channel with the starboard.
```

Comment:
these details are outdated extremely quick, cause people editing the config wont notice that its duplicated here. get rid of that. instead, describe what a starboard is.